docs/pages/database-access/guides/aws-cassandra-keyspaces.mdx

```diff
@@ -0,0 +1,201 @@
+</ScopedBlock>
+<ScopedBlock scope={["cloud"]}>
+![Teleport Database Access Redis Cloud](../../../img/database-access/guides/cassandra_keyspaces_cloud.png)
+</ScopedBlock>
+
+## Prerequisites
+
+- AWS Account with AWS Keyspaces database and permissions to create and attach IAM policies
+- cqlsh Cassandra client installed and added to your system's `PATH` environment variable.
+- A host, e.g., an Amazon EC2 instance, where you will run the Teleport Database Service.
+
+See [Installation](../../installation.mdx) for details.
+
+(!docs/pages/includes/user-client-prereqs.mdx!)
+
+
+## Step 1/5. Install and configure Teleport
+
+### Set up the Teleport Auth and Proxy Services
+
+(!docs/pages/includes/database-access/start-auth-proxy.mdx!)
+
+### Set up the Teleport Database Service
+
+(!docs/pages/includes/database-access/token.mdx!)
+
+Install Teleport on the host where you will run the Teleport Database Service:
+
+(!docs/pages/includes/install-linux.mdx!)
+
+<ScopedBlock scope={["oss", "enterprise"]}>
+
+Start the Teleport Database Service, pointing the `--auth-server` flag to the
+address of your Teleport Proxy Service:
+
+```code
+$ teleport db start \
+  --token=/tmp/token \
+  --auth-server=teleport.example.com:443 \
+  --name=keyspaces \
+  --protocol=cassandra \
+  --aws-account-id=12345678912 \
+  --aws-region=us-east-1 \
+  --labels=env=dev
+```
+
+<Admonition type="note">
+
+The `--auth-server` flag must point to the Teleport cluster's Proxy Service
+endpoint because the Database Service always connects back to the cluster over a
+reverse tunnel.
+
+</Admonition>
+
+</ScopedBlock>
+<ScopedBlock scope={["cloud"]}>
+
+Start the Teleport Database Service, pointing the `--auth-server` flag to the
+address of your Teleport Cloud tenant:
+
+```code
+$ teleport db start \
+  --token=/tmp/token \
+  --auth-server=mytenant.teleport.sh:443 \
+  --name=keyspaces \
+  --protocol=cassandra \
+  --aws-account-id=12345678912 \
+  --aws-region=us-east-1 \
+  --labels=env=dev
+```
+
+</ScopedBlock>
+
+<Admonition type="tip">
+  You can start the Database Service using a configuration file instead of CLI flags.
+  See the [YAML reference](../reference/configuration.mdx) for details.
+</Admonition>
+
+## Step 2/5. Create a Teleport user
+
+(!docs/pages/includes/database-access/create-user.mdx!)
+
+
+## Step 3/5. [Optional] Create AWS Keyspaces role:
+Create an AWS IAM Role that will be used as Keyspaces user. If you already have a custom IAM Keyspaces role, you can skip this step.
+Go to the IAM -> Access Management -> [Roles](https://console.aws.amazon.com/iamv2/home#/roles).
+Press Create Role.
+
+![Create Role Step 1](../../../img/database-access/guides/keyspaces/create-role-step1.png)
+Create custom [IAM Keyspaces Policy](https://docs.aws.amazon.com/keyspaces/latest/devguide/security_iam_id-based-policy-examples.html) or select an AWS-managed policy like `AmazonKeyspacesReadOnlyAccess` policy.
```

Comment:
```suggestion
Create a custom [IAM Keyspaces Policy](https://docs.aws.amazon.com/keyspaces/latest/devguide/security_iam_id-based-policy-examples.html) or select an AWS-managed policy like `AmazonKeyspacesReadOnlyAccess`.
```

Reply:
Should we make it more explicit that we recommend using the `AmazonKeyspacesReadOnlyAccess` policy, or which policies we recommend for different levels of access (e.g., read access or write access)? Otherwise, "an AWS-managed policy like" may be a bit too vague.

Reply:
Changed to 
```
Select `AmazonKeyspacesReadOnlyAccess` for read-only AWS Keyspaces access or `AmazonKeyspacesFullAccess` for full AWS Keyspaces access.
<Admonition type="tip">
    You can make the policy more strict by providing specific IAM role Cassandra action and specifying the Cassandra Keyspaces table [Amazon Keyspaces identity-based policy examples](https://docs.aws.amazon.com/keyspaces/latest/devguide/security_iam_id-based-policy-examples.html).
</Admonition>
```